src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    subtract(value) {
+      this.result -= value;
+
+      return this.result;
```

Comment:
The return statement in the `subtract` method is unnecessary since the `operate` method is responsible for updating the `result` property. You can remove the return statement to simplify the method.